addons/website_slides_survey/tests/test_course_certification_unlink.py

```diff
@@ -11,8 +11,8 @@ class TestSurvey(SlidesCase):
     def setUp(self):
         super(TestSurvey, self).setUp()
 
-        self.survey = self.env['survey.survey'].create({'title': 'TestSurvey'})
-        self.survey2 = self.env['survey.survey'].create({'title': 'TestSurvey'})
+        self.survey = self.env['survey.survey'].create({'title': 'TestSurvey', 'certification': True})
+        self.survey2 = self.env['survey.survey'].create({'title': 'TestSurvey', 'certification': True})
```

Comment:
Shouldn't we instead write a value into `scoring_type` (that will cascade to update "certification")?